src/components/SecurityInsightsWidget/SecurityInsightsWidget.tsx

```diff
@@ -72,7 +73,13 @@ const IssuesCounter: FC<IssuesCounterProps> = ({
   );
 };
 
-export const SecurityInsightsWidget = ({ entity }: { entity: Entity }) => {
+type Props = {
+  /** @deprecated The entity is now grabbed from context instead */
+  entity?: Entity;
+};
+
+export const SecurityInsightsWidget = (_props: Props) => {
```

Comment:
Is the exported ember named in line with the new naming conventions? perhaps should be handled separately but since we're making a major change anyway...

Reply:
yup, makes sense, I'll update the name as well

Reply:
Actually it's exported as EntitySecurityInsightsCard in `src/plugin.ts` already